pkg/txm/txm.go

```diff
@@ -216,18 +216,19 @@ func newBackoff(minDuration time.Duration) backoff.Backoff {
 	}
 }
 
-func (t *Txm) broadcastLoop(address common.Address, triggerCh chan struct{}) {
+func (t *Txm) loop(address common.Address, triggerCh chan struct{}) {
 	defer t.wg.Done()
 	ctx, cancel := t.stopCh.NewCtx()
 	defer cancel()
 	broadcastWithBackoff := newBackoff(1 * time.Second)
 	var broadcastCh <-chan time.Time
+	backfillCh := time.After(utils.WithJitter(t.config.BlockTime))
```

Comment:
`utils.WithJitter` is a [deprecated](https://github.com/smartcontractkit/chainlink-common/blob/731e805e0a2fc5351705f0c24a47c8a8719d39bd/pkg/utils/utils.go#L15) function.
It says to use `timeutils` package instead.